dashboard/pages/subscriptions.py

```diff
@@ -1,26 +1,49 @@
 """A dashboard page that lets you sign up for notifications."""
-
+from database import get_all_tags, get_connection
 import streamlit as st
+import boto3
+from os import environ as ENV
+from dotenv import load_dotenv
```

Comment:
Standard libraries (os) should come first, then third party, local.

Reply:
good spot, fixed